pkg/api/norman/customization/gke/listers.go

```diff
@@ -0,0 +1,209 @@
+package gke
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/rancher/norman/httperror"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/container/v1"
+	"google.golang.org/api/iam/v1"
+)
+
+func getOAuthClient(ctx context.Context, credentialContent string) (*http.Client, error) {
+	ts, err := google.CredentialsFromJSON(ctx, []byte(credentialContent), container.CloudPlatformScope)
+	if err != nil {
+		return nil, err
+	}
+
+	return oauth2.NewClient(ctx, ts.TokenSource), nil
+}
+
+func getComputeServiceClient(ctx context.Context, credentialContent string) (*compute.Service, error) {
+	client, err := getOAuthClient(ctx, credentialContent)
+
```

Comment:
Remove line so that error check is directly below `err` assignment. Here and other places in this file.

Reply:
Ahh, you are right. Done